exercises/variable-length-quantity/example.go

```diff
@@ -0,0 +1,15 @@
+package variablelengthquantity
```

Comment:
The example.go should be a reference solution—something that passes the given test suite. It doesn't get delivered when people `fetch` the exercise, it's just used to make sure that the test suite makes sense.